pyanaconda/ui/lib/subscription.py

```diff
@@ -86,7 +86,7 @@ def _do_payload_restart(payload):
     :param payload: Anaconda payload instance
     """
     # restart payload
-    payloadMgr.restart_thread(payload,
+    payloadMgr.start(payload,
```

Comment:
Seems that you broke parameter indentation by the rename here.

Reply:
Fixed.